apps/ima/views/declaration-immigration-adviser.html

```diff
@@ -0,0 +1,22 @@
+{{<partials-page}} {{$page-content}}
+<p class="govuk-heading-m">{{#t}}pages.declaration-immigration-adviser.paragraph1{{/t}}</p>
+<p class="govuk-body">{{#t}}pages.declaration-immigration-adviser.paragraph2{{/t}}</p>
+<div class="govuk-form-group">
+    {{#textarea}}declaration{{/textarea}}
+  </div>
+  <p class="govuk-heading-m">{{#t}}pages.declaration-immigration-adviser.header1{{/t}}</p>
+  {{#renderField}}declaration-immigration-adviser{{/renderField}} 
+ {{#markdown}}declaration-immigration-adviser{{/markdown}}
+ <br><br>
+ <div class="govuk-checkboxes__item">
+    <input class="govuk-checkboxes__input" id="declaration-immigration-adviser" name="declarationImmigrationAdviser" type="checkbox">
+    <label class="govuk-label govuk-checkboxes__label" for="declaration-immigration-adviser">
+      {{#t}}fields.declaration-immigration-adviser.label{{/t}}
+    </label>
+  </div>
+  <br><br>
+  <p>You will not be able to change your answer after you submit.</p>
```

Comment:
Should be 'You will not be able to change your answers after you submit.'.

Reply:
Changed to answers